code/WorkInProgress/ItemSpecials.dm

```diff
@@ -954,6 +954,7 @@
 		pixelaction(atom/target, params, mob/user, reach)
 			if(!isturf(target.loc) && !isturf(target)) return
 			if(!usable(user)) return
+			if(user.a_intent != "disarm") return //only want this to deploy on harm intent
```

Comment:
The comment seems to suggest this would probably be cleaner as
```suggestion
			if(user.a_intent == INTENT_HARM) return //only want this to deploy on harm intent
```
(also use intent macros)

Reply:
my comment is stupid. it should only be disarm intent. ill fix tomorrow